onnx_tf/handlers/backend/tfidf_vectorizer.py

```diff
@@ -42,7 +42,13 @@ def _calc_ngram_skip(cls, x, pool, n, skip=0):
     # Loop through the ngram targets in the pool
     tensor_list = []
     for i in range(len(pool)):
-      ngram_count = tf.map_fn(lambda in_x: tf.where(tf.reduce_all(tf.equal(in_x, tf.constant(pool[i], dtype=new_x.dtype))), tf.constant([1]), tf.constant([0])), new_x, dtype=tf.int32)
+      with tf.device("/cpu:0"):
+        ngram_count = tf.map_fn(lambda in_x: tf.where(
```

Comment:
May be add some comment to explain why need to run this map_fn on cpu device

Reply:
Thanks for the review. Added some comments. Please review again.